omnibus/config/software/datadog-agent.rb

```diff
@@ -44,14 +44,12 @@
     mkdir "#{install_dir}/scripts/"
   end
 
-  # if windows?
-  #   mkdir "../../extra_package_files/EXAMPLECONFSLOCATION"
-  #   copy "pkg/collector/dist/conf.d/*", "../../extra_package_files/EXAMPLECONFSLOCATION"
-  # end
+  if windows?
+    move 'bin/agent/dist/trace-agent.conf', "#{conf_dir}/trace-agent.conf.example"
```

Comment:
I think we want to keep moving this file on all platforms (but, on Linux, the file would get deleted in the `datadog-agent-finalize` software def)